README.md

```diff
@@ -188,10 +188,12 @@ Type following commands on a terminal to prepare a minimal project first:
 mkdir custom-fluentd
 cd custom-fluentd
 
-# Download default fluent.conf. This file will be copied to the new image.
-# VERSION is v1.4 or v0.12 like fluentd version and OS is alpine or debian.
-# Full example is https://raw.githubusercontent.com/fluent/fluentd-docker-image/master/v1.4/debian-onbuild/fluent.conf
-curl https://raw.githubusercontent.com/fluent/fluentd-docker-image/master/VERSION/OS-onbuild/fluent.conf > fluent.conf
+# Download default fluent.conf, and the . This file will be copied to the new image.
```

Comment:
What is ", and the"?

Reply:
The whole section should be like below. So please update the doc and I will close this PR.

```
# Download default fluent.conf and entrypoint.sh. This file will be copied to the new image.
# VERSION is v1.7 or v0.12 like fluentd version and OS is alpine or debian.
# Full example is https://raw.githubusercontent.com/fluent/fluentd-docker-image/master/v1.7/debian/fluent.conf

curl https://raw.githubusercontent.com/fluent/fluentd-docker-image/master/VERSION/OS/fluent.conf > fluent.conf
curl https://raw.githubusercontent.com/fluent/fluentd-docker-image/master/VERSION/OS/entrypoint.sh > entrypoint.sh
```